src/components/DropZone/DropZone.tsx

```diff
@@ -327,30 +330,36 @@ export class DropZone extends React.Component<CombinedProps, State> {
 
     this.setState({error: this.props.error});
 
-    if (!this.dropNode) {
+    const dropNode = this.dropNode.current || document;
```

Comment:
An alternative solution that may fix the problem above is user a `getter`. Rather than a `private property` we can use a dynamic value. Something like the example below may work.

```tsx
private get dropNode() {
  return this.props.dropOnPage && this.node.current ? document : null;
}
```

Reply:
That's a perfect fix! I've added that change. 😄